_episodes/03-configuration.md

```diff
@@ -3,91 +3,178 @@ title: "Configuration"
 teaching: 0
 exercises: 0
 questions:
-- "What is user configuration file and how can I use it?"
+- "What is the user configuration file and how can I use it?"
 objectives:
-- "Understand the data directories structure"
+- "Understand the contents of the user-config.yml file"
+- "Prepare the user-config.yml file for use"
 - "Configure ESMValTool to ignore some settings"
 keypoints:
-- "The ``config-user.yml`` file tells ESMValTool what data are input"
-- "The ``config-user.yml`` file tells ESMValTool what directory is the destination"
+- "The ``config-user.yml`` tells ESMValTool where to find input data."
+- " ``rootpath`` determines root directory for input data."
+- " ``output_dir`` is the destination directory."
 ---
 
 ## The configuration file
 
 The ``config-user.yml`` configuration file contains all the global level information needed by ESMValTool to run.
-This is an (YAML file) [https://yaml.org/spec/1.2/spec.html]. An example configuration file can be found in the root directory of the ESMValTool repository.
-Make a copy and rename it to ``config-user.yml``:
+This is an [YAML file](https://yaml.org/spec/1.2/spec.html). An example configuration file can be found in the root directory of the ESMValTool repository: [config-user-example.yml](https://github.com/ESMValGroup/ESMValTool/blob/master/config-user-example.yml).
 
+Let's download it to our working directory ``esmvaltool_tutorial`` that is made during the [Setup](https://esmvalgroup.github.io/tutorial/setup.html).
```

Comment:
Maybe rephrase to "... that was created during the [Setup]..."?

Reply:
It might be good to create the link as described in the section [Locations and Names](https://carpentries.github.io/lesson-example/04-formatting/index.html), because then it will still work even if the repository is moved/renamed.